### ISSUE TITLE
Title in Hero Card does not have `aria-level` specified

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4327]
## Screenshots

![image](https://user-images.githubusercontent.com/1622400/175499104-6af48e58-7973-4668-bf4b-c4c76f19f258.png)

## Version

`main`

## Describe the bug

In Adaptive Cards library, by default, all `TextBlock` with `style="heading"` will have `role="heading"` set. However, the required attribute, `aria-level`, was not set. To set the `aria-level`, we need to *explicitly* specify `hostConfig.textBlock.headingLevel`.

Adaptive Cards library added this new host config option in their https://github.com/microsoft/AdaptiveCards/issues/5635 and https://github.com/microsoft/AdaptiveCards/pull/5698.

Without specifying this new host config option, the rendered DOM tree will be invalid by WCAG 4.1.2 (aria-required-attr). This is because all elements with `role="heading"` MUST have `aria-level` specified.

## Steps to reproduce

1. Talk to MockBot
2. Say "herocard"
3. Inspect the element "Details about image 1"

## Expected behavior

The element has `role="heading"`, however, it does not have `aria-level` attribute specified.

## Action items

Make sure to update our `validateDOMTree()` function to verify this issue on a broader scale.

## Additional context

New bug https://github.com/microsoft/AdaptiveCards/issues/7566 filed to `microsoft/AdaptiveCards` repo.

- https://github.com/microsoft/AdaptiveCards/issues/5635
- https://github.com/microsoft/AdaptiveCards/pull/5698

[Bug]
